### ISSUE TITLE
Integrate Rhubarb Lip Sync with Tahoma

### ISSUE THREAD

**Issue by artisteacher** [turtletooth/tahoma2d#270]
**Is your feature request related to a problem? Please describe**
Tahoma uses .dat files for lip syncing. It would be amazing if the .dat files could be generated within Tahoma itself  - instead of needing to rely on a 3rd party app. The original Papagayo hasn't run well on macOS in years. Papagayo NG doesn't have a macOS release.  Rhubarb Lip Sync analyzes a track & automatically generates a .dat file. It works well, but requires command line knowledge - not the best fit for many students.

**Describe the solution you'd like**
If it's possible, it would be amazing to integrate [Rhubarb Lip Sync](https://github.com/DanielSWolf/rhubarb-lip-sync) directly with Tahoma. I'd integrate it with the "Apply Lip Sync Data" window. There would need to be a new field to choose the audio track, an optional "load script" field, & a button to analyze & apply. 

There should still be a way to load a .dat file manually, for anyone that prefers that workflow. I'd probably add buttons that could switch between an "auto mode" (with Rhubarb) and a "manual mode" (current behavior), directly in the window.

**Comment by jeremybullock** [comment]
This is planned for 1.2

**Comment by ladlon** [comment]
@JeremyBullock:  That's awesome news!

I tried Papagayo for the first time, when switching over to Tahoma2D about a week ago... and as much as I like what it does, I found it has some (kind of painful) aspects... like common mouse controls and other things we are generally used to.  The interface/workflow feels pretty clunky and 'old' in some respects.  Simple things like moving/scaling the waveform display, selecting/moving elements, and playing back of specific sections were a bit clunky, and sometimes went against the standard ways you would expect them to be done/controlled.

Hopefully an internal lipsync system in Tahoma2D would have a more modern/steamlined control system.  I just found the existing Papagayo control system a bit of a frustration, and unnecessarily slowing down the whole process.  I can't knock any free software, and certainly appreciate what it IS capable of, but it certainly something I'd love to see improved on.
